packages/tailwindcss/src/utilities.ts

```diff
@@ -1300,6 +1300,7 @@ export function createUtilities(theme: Theme) {
       let value
       if (candidate.value.kind === 'arbitrary') {
         value = candidate.value.value
+        value = negative ? `calc(${value} * -1)` : value
```

Comment:
Seems like this needs a changelog entry

Reply:
Ah yep! I noticed this when one of the migrations felt off:

<img width="862" alt="image" src="https://github.com/user-attachments/assets/7e001bdd-a485-45b1-8106-a7342d4e13e7" />

Spot the bug 👀